### ISSUE TITLE
eclipse+RED Robot Editor，False positive error

### ISSUE THREAD

**Issue by zhoujimin** [nokia/RED#428]
eclipse+RED Robot Editor，False positive error，How to solve，thanks
![image](https://user-images.githubusercontent.com/67677349/128338835-008227e2-21e5-4d1c-8c5a-cb1947777e22.png)

**Comment by michalanglart** [comment]
Yes, this is known: you can have a look into #390 and issues which are mentioned there by me. Then also https://github.com/nokia/RED/issues/195#issuecomment-375589938 for deeper explanation where you can find also links to workaounds.

Whether this one is false positive or not depend on how you run the tests. If you would run only the second test you will have an error too because it depends on the first one.
